src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -104,7 +95,7 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
        |        "S": "$batchId.json"
        |      },
        |      "type": {
-       |        "S": "ArchiveFolder"
+       |        "S": "ContentFolder"
```

Comment:
Just curious, how come you changed this?

Reply:
Don't know. Bit of variety. :smile: I'll leave it as it shows that ArchiveFolder and ContentFolder will work.